internal/errors/blob_test.go

```diff
@@ -313,3 +285,143 @@ func TestIsErrBlobNoSuchKey(t *testing.T) {
 		})
 	}
 }
+
+func TestErrBlobNoSuchBucket_Unwrap(t *testing.T) {
+	type fields struct {
+		err error
+	}
+	type want struct {
+		err error
+	}
+	type test struct {
+		name       string
+		fields     fields
+		want       want
+		checkFunc  func(want, error) error
+		beforeFunc func()
+		afterFunc  func()
+	}
+	defaultCheckFunc := func(w want, err error) error {
+		if !Is(err, w.err) {
+			return Errorf("got_error: \"%#v\",\n\t\t\t\twant: \"%#v\"", err, w.err)
+		}
+		return nil
+	}
+	tests := []test{
+		func() test {
+			err := New("internal error")
+			return test{
+				name: "returns the unwrapped error",
+				fields: fields{
+					err: err,
+				},
+				want: want{
+					err: err,
+				},
+			}
+		}(),
+		{
+			name: "returns the unwrapped nil error",
```

Comment:
```suggestion
			name: "returns nil when get the unwrapped nil error",
```

Reply:
I think it is more correct.
```suggestion
			name: "returns nil when getting the unwrapped nil error",
```